share/getters/single_eds.go

```diff
@@ -0,0 +1,51 @@
+package getters
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/celestiaorg/celestia-app/pkg/da"
+	"github.com/celestiaorg/celestia-node/share"
+	"github.com/celestiaorg/nmt/namespace"
+	"github.com/celestiaorg/rsmt2d"
+)
+
+// SingleEDSGetter contains single EDS and allows getting data only out of it.
+type SingleEDSGetter struct {
```

Comment:
Is it going to be used outside of testing purposes? If no, I would vote for creating subpkg `getterstest` to reduce import surface and API of main getters pkg

Reply:
Very likely not to be used anywhere else but made reusable not only in testing by purpose. Agree thought that we should move it elsewhere and we should keep this post-implementation chores.